### DISCUSSION TITLE
I can't obtain a validation of a "Spatial Data Service" metadata

### DISCUSSION THREAD

**Discussion by pdorio** [INSPIRE-MIF/helpdesk-validator#799] [Q&A] [answered]
I have been trying for two days to modify my XML to obtain full validation for my metadata of type 'Spatial Data Service'.

I attach [my XML](https://github.com/INSPIRE-MIF/helpdesk-validator/files/8991687/service.txt). I would like to be able to get a complete valid metadata, i.e. for Conformance Classes 3-5-6-7

Here the result report:
https://inspire.ec.europa.eu/validator/test-reports/details.html?id=EID0a7c3cfa-3937-4d4b-bea2-b8df6c4453ed

most of the problems are on the 'data quality' part but also on the 'invocation' where it asks me for the access point description to be 'accessPoint' and that's it. I just don't understand, I did copy-paste the blocks from the TD-MD 2.0 specification

can someone help me?

Regards, Pietro

**Comment by fabiovinci** [comment, marked as answer]
Dear @pdorio,

what type of service are you writing metadata for?

Looking at your sample, it seems for WMS/WFS. If so, please consider that they are "Network Services" and not generic "Spatial Data Services", so you need to comply with Conformance Classes 3-4.

You can find some example metadata here: https://github.com/INSPIRE-MIF/helpdesk-validator/tree/master/examples

**Reply by pdorio** [reply]
Thank you very much @fabiovinci! For which kind of Services must be use the "Spatial Data Services" validation?

**Reply by fabiovinci** [reply]
There is no specific definition of generic "Spatial Data Services", for sure you can exclude those categorized as "Network Services", i.e. specified in Regulation 976/2009.

See section [1.1.2. Categories of INSPIRE Spatial Data Services](https://github.com/INSPIRE-MIF/technical-guidelines/blob/main/metadata/metadata-iso19139/metadata-iso19139.adoc#112-categories-of-inspire-spatial-data-services) in the MD TG for more reference.

**Comment by pdorio** [comment]
Dear @fabiovinci now i finally become a "passed manual" validation for my service.

I will obtain a clean passed validation
Do you have any suggestions?

[Here my final XML](https://github.com/INSPIRE-MIF/helpdesk-validator/files/9020319/service.txt)

Here my last test report: https://inspire.ec.europa.eu/validator/test-reports/details.html?id=EID3eeac3e2-2e1a-40b4-9e6b-9284a488ec33

**Comment by fabiovinci** [comment]
Dear @pdorio,

since it seems the report is not accessible, I re-validate your metadata (https://inspire.ec.europa.eu/validator/test-run/details.html?id=EIDb9ec8519-2f3b-4694-a6d1-06a1df77ff93) and there is only one error related to the "Coupled Resource" element.

> Do you have any suggestions?

Are you referring to this one?

**Comment by pdorio** [comment]
Ok, i tried another time.

Here the final report "PASSED MANUAL"
https://inspire.ec.europa.eu/validator/test-run/details.html?id=EID54e9a962-e687-41c9-8737-1cd42e5477a6

I can't understand how to fix this:
_"The metadata record set has 1 record(s) with errors for this assertion.
XML document 'p_bz_Elevation.xml', record 'p_bz:Elevation': Check manually that a spatial resolution restriction is provided being mandatory when there is a restriction on the spatial resolution for the service."_

[Here my final XML](https://github.com/INSPIRE-MIF/helpdesk-validator/files/9046393/p_bz_Elevation.txt)

**Comment by fabiovinci** [comment]
Dear @pdorio,

the multiplicity of the [related metadata element](https://github.com/INSPIRE-MIF/technical-guidelines/blob/main/metadata/metadata-iso19139/metadata-iso19139.adoc#4121-restriction-on-the-spatial-resolution), as defined in [Regulation 1205/2008], Part C, Table 2, is zero or more, with the following condition: "Mandatory when there is a restriction on the spatial resolution for this service".

Since these restrictions shall be expressed in the gmd:abstract element with a Non-empty Free Text Element content, there is no possibility to test it in an automatic way, and the test is manual.
